.github/pull_request_template.md

```diff
@@ -1,14 +1,11 @@
-Changes proposed in this PR:
--
--
+### Changes proposed in this PR:
 
-How I've tested this PR:
+### How I've tested this PR:
 
-How I expect reviewers to test this PR:
+### How I expect reviewers to test this PR:
```

Comment:
Happy to see this! I just went through and bolded my own yesterday

Reply:
Saaaame, I've been doing that for all my PRs manually haha